myhpom/templates/myhpom/accounts/next_steps_no_ad_template.html

```diff
@@ -41,18 +41,22 @@ <h3>Welcome!</h3>
                             You may still store your documents with us
                             and retrieve them upon request.
                         </p>
-                        <form>
+                        <form method='post'>
```

Comment:
This doesn't work for me when I test manually - couple things I'm noticing: the submit button is not in the form (and  has an href for 'home'), and this form doesn't have an action.